tests/test_utils.py

```diff
@@ -61,7 +61,7 @@ def test_get_extended_access_token():
 def test_get_extended_access_token_v23_plus():
     mock_request.return_value.status_code = 200
     mock_request.return_value.content = (
-        '{"access_token":"<extended access token>","token_type":"bearer"}'
+        '{"access_token":"<extended access token>","token_type":"bearer", "expires_in":5170982}'
```

Comment:
Hello @jgorset..Strange 😅 , mine also returned with `expires_in` key, the same as @Badzyo 's, maybe there is something to do with region or app configuration. Let me try to investigate more.